wiring/inc/spark_wiring_cellular.h

```diff
@@ -74,6 +74,12 @@ class CellularClass : public NetworkClass
 
     CellularSignal RSSI();
 
+    CellularData dataUsage(void);
+
+    CellularData dataUsage(const CellularData &data_set);
```

Comment:
This should return a `CellularData&` or it's going to create a new copy of CellularData, which I'm guessing would be counter-intuitive.